problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -79,7 +79,15 @@ function MentoringWithStepsBlock(runtime, element) {
         $.post(handlerUrl, JSON.stringify({}))
             .success(function(response) {
                 attemptsDOM.data('num_attempts', response.num_attempts);
-                // Now that relevant info is up-to-date, get the latest grade
+                publishAttempt();
+            });
+    }
+
+    function publishAttempt() {
+        var handlerUrl = runtime.handlerUrl(element, 'publish_attempt');
+        $.post(handlerUrl, JSON.stringify({}))
```

Comment:
@e-kolpakov Logic for publishing `"grade"` event is now covered by the `test_step_builder` integration test.